packages/mobile/locales/en-US/onboarding.json

```diff
@@ -38,7 +38,9 @@
     "label": "Invite Code",
     "codePlaceholder": "Enter invite code",
     "noCode": "Don’t have an Invite Code",
-    "nodeCodeInviteLink": "Don’t have an invite code? Request one at <0>celo.org/developers/wallet</0> or <1>continue without</1>"
+    "nodeCodeInviteLink": "Don’t have an invite code? Request one at <0>celo.org/developers/wallet</0> or <1>continue without</1>",
```

Comment:
are we still wanting to have this in the public build? its a dev only feature that doesnt even work right now

Reply:
Thanks for checking.
This is controlled by the `SHOW_GET_INVITE_LINK` config, which is appropriately set to `false` for public builds.